app/screens/InfoScreen/CreditsScreen.tsx

```diff
@@ -0,0 +1,157 @@
+import React from "react"
+import { Dimensions, TextStyle, View, ViewStyle } from "react-native"
+import { Screen, Text } from "../../components"
+import { translate } from "../../i18n"
+import { useAppNavigation, useHeader } from "../../hooks"
+import { colors, spacing } from "../../theme"
+import { FlashList } from "@shopify/flash-list"
+
+export const CreditsScreen = () => {
+  const navigation = useAppNavigation()
+  const { width } = Dimensions.get("window")
+
+  useHeader({
+    title: translate("infoScreen.creditsTitle"),
+    leftIcon: "back",
+    onLeftPress: () => navigation.goBack(),
+  })
+
+  type Person = {
+    name: string
+    role: string
+  }
+
+  const data: Person[] = [
+    {
+      name: "Gant Laborde",
+      role: "Organizer of Chain React",
+    },
+    {
+      name: "Justin Huskey",
+      role: "Co-organizer of Chain React & App Design",
+    },
+    {
+      name: "Jamon Holmgren",
+      role: "Co-organizer of Chain React & App Development",
+    },
+    {
+      name: "Jenna Fucci",
+      role: "Designer (web, app, and print)",
+    },
+    {
+      name: "Simran Sachdeva",
+      role: "CFP Committee & Workshop Assistant",
```

Comment:
she's also a tester

Reply:
Updated in [0caad88](https://github.com/infinitered/ChainReactApp2023/pull/154/commits/0caad88a06c41933d07e95091cfc1c9ca3c3b5cc)